lib/stitches/api_generator.rb

```diff
@@ -4,22 +4,30 @@ module Stitches
   class ApiGenerator < Rails::Generators::Base
     include Rails::Generators::Migration
 
-    source_root(File.expand_path(File.join(File.dirname(__FILE__),"generator_files")))
+    source_root(File.expand_path(File.join(File.dirname(__FILE__), "generator_files")))
 
     def self.next_migration_number(path)
       Time.now.utc.strftime("%Y%m%d%H%M%S")
     end
 
     desc "Bootstraps your API service with a basic ping controller and spec to ensure everything is setup properly"
     def bootstrap_api
-      inject_into_file "Gemfile", after: /^gem ['"]rails['"].*$/ do<<-GEM
-
-gem "apitome"
-gem "responders"
-gem "rspec_api_documentation", group: [ :development, :test ]
-gem "capybara", group: [ :development, :test ]
```

Comment:
Looks like you dropped capybara along the way. Intentional I assume because stitches has no frontend?

Reply:
Correct, capybara isn't actually a requirement.